packages/web-components/fast-components/src/menu-item/menu-item.styles.ts

```diff
@@ -5,6 +5,7 @@ import {
     focusVisible,
     forcedColorsStylesheetBehavior,
 } from "@microsoft/fast-foundation";
+import { SystemColors } from "@microsoft/fast-web-utilities";
```

Comment:
Do you know what is changing this? I want to be sure that this will not change in the future, we had an issue with the generated palette constantly being updated on PRs due to a prettier version mismatch.

Reply:
This was a mistake by me, I was at one time using this component as a go by and changes got included here accidentally. will remove